server/common/annotations/hosted_tiledb.py

```diff
@@ -21,6 +22,13 @@ def __init__(self, directory_path, db):
         super().__init__()
         self.db = db
         self.directory_path = directory_path
+        config = tiledb.Config()
+        config["vfs.s3.region"] = "us-west-2"
+        self.ctx = tiledb.Ctx(config=config)
+        try:
+            tiledb.default_ctx(config)
+        except TileDBError:
+            pass
```

Comment:
Don't hard-code the region.

There is already a config for tiledb in the CxgAdaptor which will have the correct region set (along with other configuration options).   Please set that to be the default_ctx in the set_tiledb_context function of CxgAdaptor in server/data_cxg/cxg_adaptor.py.  Then you can remove lines 25-31

Reply:
it looks like it is currently hard coded to a different region in the cxg adaptor https://github.com/chanzuckerberg/cellxgene/blob/bc150a84698cf7a28900df865ad9daea85dc0c1c/server/data_cxg/cxg_adaptor.py#L25

Reply:
discussed over slack